ivy/functional/ivy/elementwise.py

```diff
@@ -54,6 +54,76 @@ def abs(
         an array containing the absolute value of each element in ``x``. The returned
         array must have the same data type as ``x``.
 
+    Functional Examples
+    -------------------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([-1,0,-6])
+    >>> y = ivy.abs(x)
+    >>> print(y)
+    ivy.array([1, 0, 6])
+
+    >>> x = ivy.array(3.7, -7.7, 0, -2, -0])
+    >>> y = ivy.abs(x)
+    >>> print(y)
+    ivy.array([ 3.7, 7.7, 0, 2, 0])
+
+    >>> x = ivy.array([[1.1, 2.2, 3.3], [-4.4, -5.5, -6.6]])
+    >>> ivy.abs(x, out=x)
+    >>> print(x)
+    ivy.array([[ 1.1,  2.2,  3.3],
+               [4.4, 5.5, 6.6]])
+               
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([0, -0, -2.6, -1, 1, 3.6])
+    >>> y = ivy.abs(x)
+    >>> print(y)
+    ivy.array([ 0, 0, 2.6, 1, 1, 3.6])
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([0., 2.6, -3.5]),\
```

Comment:
Please update the formatting here, so that `...` is removed and the second line is positioned in a way that it aligns with `a` above it.